marklogic-unit-test-client/src/test/ml-modules/root/test/suites/Assertions/assert-multi-statement.xqy

```diff
@@ -0,0 +1,22 @@
+import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";

```

Comment:
Just to confirm - this test fails if the Jaxp class above isn't modified?

Reply:
Yes, this test fails without jaxp class modification